netmaster/objApi/apiController.go

```diff
@@ -657,6 +657,57 @@ func (ac *APIController) EndpointGroupUpdate(endpointGroup, params *contivModel.
 	return nil
 }
 
+// EndpointGroupGetOper inspects endpointGroup
+func (ac *APIController) EndpointGroupGetOper(endpointGroup *contivModel.EndpointGroupInspect) error {
+	log.Infof("Received EndpointGroupInspect: %+v", endpointGroup)
+
+	// Get the state driver
+	stateDriver, err := utils.GetStateDriver()
+	if err != nil {
+		return err
+	}
+
+	epgCfg := &mastercfg.EndpointGroupState{}
+	epgCfg.StateDriver = stateDriver
+	epgID := endpointGroup.Config.GroupName + ":" + endpointGroup.Config.TenantName
+	fmt.Printf("EPGID is %s", epgID)
+	if err := epgCfg.Read(epgID); err != nil {
+		log.Errorf("Error fetching endpointGroup from mastercfg: %s", epgID)
+		return err
+	}
+
+	endpointGroup.Oper.ExternalPktTag = epgCfg.ExtPktTag
+	//network.Oper.NumEndpoints = nwCfg.EpCount
```

Comment:
We should populate this field based on number of endpoints in endpointGroup.Oper.Endpoints array